packages/portal/spec-renderer/src/components/SpecOperationsList.vue

```diff
@@ -119,6 +130,7 @@ import type { OperationListFilterFunction, Operation, OperationListItem, Tag } f
 import { v1 as uuidv1 } from 'uuid'
 import clonedeep from 'lodash.clonedeep'
 import composables from '../composables'
+import { KInput, KIcon, KCollapse } from '@kong/kongponents'
```

Comment:
issue: this needs to be removed; Kongponents are available within the host app. If they aren't working in the tests, then the test setup needs to be updated instead.